src/test/clip.t.sol

```diff
@@ -1138,6 +1173,65 @@ contract ClipperTest is DSTest {
         assertTrue(try_redo(1, address(this)));
     }
 
+    function test_redo_incentive() public takeSetup {
+        // (, uint256 price) = clip.getStatus(1);
+        // assertEq(price, ray(5 ether));
+
+        clip.file("tip",  rad(100 ether)); // Flat fee of 100 DAI
+        clip.file("chip", 0);              // No linear increase
+
+        (, uint256 tab, uint256 lot,,,) = clip.sales(1);
+
+        assertEq(tab, rad(110 ether));
+        assertEq(lot, 40 ether);
+
+        hevm.warp(now + 300);
+        clip.redo(1, address(123));
+        assertEq(vat.dai(address(123)), clip.tip());
+
+        clip.file("chip", 0.02 ether);     // Linear increase of 2% of tab
+        hevm.warp(now + 300);
+        clip.redo(1, address(234));
+        assertEq(vat.dai(address(234)), clip.tip() + clip.chip() * tab / WAD);
+
+        clip.file("tip", 0); // No more flat fee
+        hevm.warp(now + 300);
+        clip.redo(1, address(345));
+        assertEq(vat.dai(address(345)), clip.chip() * tab / WAD);
+
+        vat.file(ilk, "dust", rad(110 ether) + 1); // 1 wei > than $110 (tab) as dust
+
+        hevm.warp(now + 300);
+        clip.redo(1, address(456));
+        assertEq(vat.dai(address(456)), 0);
+
+        vat.file(ilk, "dust", rad(20 ether)); // $20 dust
```

Comment:
add comment here explaining that you are testing the dusty lot case

Reply:
done!